DMFT/.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
```

Comment:
Add `*.env` files.
They are a common standard for configuring environment variables on each developer box.
They are also commonly used for docker.